wdl2cwl/main.py

```diff
@@ -549,24 +548,66 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:
             only_arg = arguments[0]
             return self.get_expr(only_arg)
+        elif function_name == "select_first":
+            array_obj = arguments[0]
+            array_items = [self.get_expr(item) for item in array_obj.items]
+            return f"{array_items}.find(mem => mem !== null) "
+        elif function_name == "_mul":
+            left_operand, right_operand = arguments
+            left_str = self.get_expr(left_operand)
+            right_str = self.get_expr(right_operand)
+            return f"{left_str}*{right_str}"
+        elif function_name == "_eqeq":
+            left_operand, right_operand = arguments
+            left_str = self.get_expr(left_operand)
+            right_str = self.get_expr(right_operand)
+            return f"{left_str} === {right_str}"
+        elif function_name == "ceil":
+            only_arg = self.get_expr(arguments[0])
+            return f"Math.ceil({only_arg}) "
+        elif function_name == "_div":
+            left_operand, right_operand = arguments
+            left_str = self.get_expr(left_operand)
+            right_str = self.get_expr(right_operand)
+            return f"{left_str}/{right_str}"        
+        elif function_name == "size":
+            left_operand, right_operand = arguments
+            left_str = self.get_expr(left_operand)
+            size_unit = self.get_expr(right_operand)[1:-1]
+            with WDLSourceLine(size_unit, ConversionException):
+                if size_unit == "Ki" or size_unit == "K":
+                    unit_value = ""
+                elif size_unit == "Mi" or size_unit == "M":
+                    unit_value = 2
+                if size_unit == "Gi" or size_unit == "G":
+                    unit_value = 3
+                elif size_unit == "Ti" or size_unit == "T":
+                    unit_value = 4
+                elif size_unit == "Pi" or size_unit == "P":
+                    unit_value = 5
+                else:
+                    raise WDLSourceLine(size_unit, ConversionException).makeError(
+                        f"Size Unit '{size_unit}' not yet handled."
+                    )
+            return f"{left_str}.size / 1024^{unit_value}"
 
         raise WDLSourceLine(wdl_apply_expr, ConversionException).makeError(
             f"Function name '{function_name}' not yet handled."
         )
 
     def get_expr_get(self, wdl_get_expr: WDL.Expr.Get) -> str:
         """Translate WDL Get Expressions."""
-        with WDLSourceLine(wdl_get_expr, ConversionException):
-            member = wdl_get_expr.member
-            if (
-                not member
-                and isinstance(wdl_get_expr.expr, WDL.Expr.Ident)
-                and wdl_get_expr.expr
-            ):
-                return self.get_expr_ident(wdl_get_expr.expr)
-            raise ConversionException(
-                f"Get expressions with {member} are not yet handled."
-            )
+        member = wdl_get_expr.member
```

Comment:
haneld get expressions with member. These members are identities contained in custom types called struct in WDL